### PR TITLE
Bump applicable nuget packages to latest versions

### PR DESCRIPTION
**Pull request by taooceros**
#486 
We cannot bump squirrel higher than 1.9.0

### CODE REVIEW

**Comment by taooceros** [comment]
I didn't find some issue currently. Though, our pinyin library has updated its data source, which seems significantly increase the memory usage.

**Comment by jjw24** [comment]
> our pinyin library has updated its data source, which seems significantly increase the memory usage

cpu and memory usage is important, so if the library increases memory, let's not bump it.

File: Flow.Launcher.Infrastructure/Flow.Launcher.Infrastructure.csproj [hunk 1]
```diff
@@ -53,9 +53,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.10.56" />
     <PackageReference Include="NLog" Version="4.7.10" />
     <PackageReference Include="NLog.Schema" Version="4.7.10" />
-    <PackageReference Include="NLog.Web.AspNetCore" Version="4.12.0" />
-    <PackageReference Include="System.Drawing.Common" Version="4.7.0" />
-    <PackageReference Include="ToolGood.Words.Pinyin" Version="3.0.1.4" />
+    <PackageReference Include="NLog.Web.AspNetCore" Version="4.13.0" />
+    <PackageReference Include="System.Drawing.Common" Version="5.0.2" />
+    <PackageReference Include="ToolGood.Words.Pinyin" Version="3.0.2.6" />
```
Comment (jjw24) [inline Flow.Launcher.Infrastructure/Flow.Launcher.Infrastructure.csproj:58]: should we not bump this then

**Comment by taooceros** [comment]
The memory inflation will only appear when people use pinyin, but the new version can also increase the performance of pinyin, which probably is acceptable.

**Comment by jjw24** [comment]
> downgrade pinyin library until further discussion

Thanks, i was about to suggest to pull this out as a seperate PR. Did you find more issues from this bump?

**Comment by jjw24** [comment]
havent had the time to look into this, we are getting error with the build:
image

**Comment by jjw24** [comment]
Ok so this error is coming with nuget pack version > 5.8.1. This is part of the wider .net 5 sdk change now requiring you to specify platform version e.g. net5.0-windowsX.Y (X.Y as TPV- TargetPlatformVersion). The target framework net5.0-windows for some of our projects is still valid in the csproj files even though we are not specifying which version of Windows because the .NET SDK provides a default, and NuGet uses that to generate lib/net5.0-windows7.0/your.dll. Since we are not packing via csproj, we wont get the message from there, but we are using command-line to pack so need to append the platform version number to our nuspec.

I am going to separate out this issue so I can isolate the testing for this as I also want to try appending the target platform version to the csproj files.

**Review by jjw24 — DISMISSED** [review]

**Review by jjw24 — APPROVED** [review]

**Pull request merged**